README.md

```diff
@@ -48,6 +48,7 @@ This is the JavaScript version of [OpenTelemetry](https://opentelemetry.io/), a
 
 | Core version | Contrib Version         |
 |--------------|-------------------------|
+| 0.17.x       | 0.14.x not released yet |
 | 0.16.x       | 0.14.x not released yet |
```

Comment:
Are we sure that 0.14.x plugins will work with 0.16 *and* 0.17?

Reply:
we haven't released 0.14 yet but we already have version 0.16 and 0.17, once we finally release 0.14 we can fix it, the version 0.16 will be an "orphan" then. It may also happen we will have next version of core 0.18 and still not contrib repo but we should at least indicate that version 0.16 and 0.17 is not compatible with any contrib version

Reply:
As far as I know it's already clear now that we will never have a contrib release for 0.16.x so why not just writing this here already now?